code/modules/uplink/uplink_items/stealthy_tools.dm

```diff
@@ -0,0 +1,94 @@
+// File ordered based on progression
+
+/datum/uplink_category/stealthy_tools
+	name = "Stealth Gadgets"
+	weight = 4
+
+/datum/uplink_item/stealthy_tools
+	category = /datum/uplink_category/stealthy_tools
+
+// No progression cost
+
+/datum/uplink_item/stealthy_tools/agent_card
+	name = "Agent Identification Card"
+	desc = "Agent cards prevent artificial intelligences from tracking the wearer, and hold up to 5 wildcards \
+			from other identification cards. In addition, they can be forged to display a new assignment, name and trim. \
+			This can be done an unlimited amount of times. Some Syndicate areas and devices can only be accessed \
+			with these cards."
+	item = /obj/item/card/id/advanced/chameleon
+	cost = 2
+
+/datum/uplink_item/stealthy_tools/ai_detector
+	name = "Artificial Intelligence Detector"
+	desc = "A functional multitool that turns red when it detects an artificial intelligence watching it, and can be \
+			activated to display their exact viewing location and nearby security camera blind spots. Knowing when \
+			an artificial intelligence is watching you is useful for knowing when to maintain cover, and finding nearby \
+			blind spots can help you identify escape routes."
+	item = /obj/item/multitool/ai_detect
+	cost = 1
+
+/datum/uplink_item/stealthy_tools/chameleon
+	name = "Chameleon Kit"
+	desc = "A set of items that contain chameleon technology allowing you to disguise as pretty much anything on the station, and more! \
+			Due to budget cuts, the shoes don't provide protection against slipping and skillchips are sold separately."
+	item = /obj/item/storage/box/syndie_kit/chameleon
+	cost = 2
+	purchasable_from = ~UPLINK_NUKE_OPS //clown ops are allowed to buy this kit, since it's basically a costume
+
+/datum/uplink_item/stealthy_tools/chameleon_proj
+	name = "Chameleon Projector"
+	desc = "Projects an image across a user, disguising them as an object scanned with it, as long as they don't \
+			move the projector from their hand. Disguised users move slowly, and projectiles pass over them."
+	item = /obj/item/chameleon
+	cost = 7
+
+/datum/uplink_item/stealthy_tools/codespeak_manual
+	name = "Codespeak Manual"
+	desc = "Syndicate agents can be trained to use a series of codewords to convey complex information, which sounds like random concepts and drinks to anyone listening. \
+			This manual teaches you this Codespeak. You can also hit someone else with the manual in order to teach them. This is the deluxe edition, which has unlimited uses."
+	item = /obj/item/language_manual/codespeak_manual/unlimited
+	cost = 3
+
+/datum/uplink_item/stealthy_tools/emplight
```

Comment:
it's good that this is available as it at least protects somewhat from borgs and allows for you to discreetly disable headsets. It isn't quite the same as the EMP kit but definitely should be available at this point, so I agree with this.